datafusion/groupby-datafusion.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+
+print("# groupby-datafusion.py", flush=True)
+
+import os
+import gc
+import timeit
+import datafusion as df
+from datafusion import functions as f
+from datafusion import col
+from pyarrow import csv as pacsv
+
+exec(open("./_helpers/helpers.py").read())
+
+def ans_shape(batches):
+    rows, cols = 0, 0
+    for batch in batches:
+        rows += batch.num_rows
+        if cols == 0:
+            cols = batch.num_columns
+        else:
+            assert(cols == batch.num_columns)
+    
+    return rows, cols
+
+ver = df.__version__
+git = ""
+task = "groupby"
+solution = "datafusion"
+fun = ".groupby"
+cache = "TRUE"
+on_disk = "FALSE"
+
+data_name = os.environ["SRC_DATANAME"]
+src_grp = os.path.join("data", data_name + ".csv")
+print("loading dataset %s" % data_name, flush=True)
+
+data = pacsv.read_csv(src_grp, convert_options=pacsv.ConvertOptions(auto_dict_encode=True))
+
+ctx = df.ExecutionContext()
```

Comment:
The latest version of data fusion seems to be 26.0.0. If I try to run the benchmark locally I get the following error on this line
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'datafusion' has no attribute 'ExecutionContext
```